server/models/Task.js

```diff
@@ -56,7 +56,30 @@ export default class Task extends Model {
       description: { type: 'string', minLength: 3 },
       statusId: { type: 'integer' },
       creatorId: { type: 'integer' },
-      assignedId: { type: ['integer', 'null'] },
+      assignedId: {
+        anyOf: [
+          { type: 'integer' },
+          { type: 'null' },
+        ],
+      },
+    },
+  };
+
+  static modifiers = {
+    withLabelIn(query, labelIds) {
+      if (labelIds.length) {
+        query.where('labels.id', 'in', labelIds);
+      }
+    },
+    withStatusIn(query, statusIds) {
+      if (statusIds.length) {
+        query.where('status_id', 'in', statusIds);
+      }
+    },
+    withAssignedIn(query, assignedIds) {
+      if (assignedIds.length) {
+        query.where('assigned_id', 'in', assignedIds);
+      }
```

Comment:
Cool, I did the same thing when I first used modifiers. I'm talking about if inside modifers. Unfortunately it's considered as a code smell. Because now modifier is in control of your query behaviour.